spec/custom-runner.js

```diff
@@ -0,0 +1,22 @@
+const { createRunner } = require('atom-jasmine3-test-runner')
+
+const options = {
+  specHelper: {
+    atom: true,
+    attachToDom: true,
+    ci: true,
+    customMatchers: true,
+    jasmineFocused: true,
+    jasmineJson: true,
+    jasminePass: true,
+    jasmineTagged: true,
+    mockClock: true,
+    mockLocalStorage: true,
+    profile: true,
+    set: true,
+    unspy: true
+  },
```

Comment:
There is a shorter version
```suggestion
  specHelper: true,
```